src/main/java/pl/stormit/tinyurl/domain/model/UrlExpiry.java

```diff
@@ -0,0 +1,32 @@
+package pl.stormit.tinyurl.domain.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.time.Instant;
+import java.util.UUID;
+
+@Entity
+@Table(name = "url_expiry")
+@Getter
+@Setter
+@ToString
+public class UrlExpiry {
+
+    @Id
+    @GeneratedValue
+    private UUID id = UUID.randomUUID();
+
+    @Column(nullable = false)
+    private Instant creationDate = Instant.now();
+
+    private Instant expirationDate;
+
+    @Column(nullable = false)
+    private Boolean isPremium = false;
```

Comment:
I'm not fully convinced, that it's a good idea. 
We will mix functionalities and their logic. 
Maybe for premium users, we should, for example not create expire at all?
For now, It can stay, but think about it.